File: 07-Problem-Variables/07-Problem-Variables-CPP/07-Problem-Variables.cpp
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

#include <iostream>
using namespace std;

int main() {
    int myMutable = 10;
    const int MY_IMMUTABLE = 19;

    cout << "myMutable: " << myMutable << " MY_IMMUTABLE: " << MY_IMMUTABLE << endl;


    myMutable += MY_IMMUTABLE;

    cout << "myMutable: " << myMutable << " MY_IMMUTABLE: " << MY_IMMUTABLE << endl;

    return 0;
}

File: 07-Problem-Variables/07-Problem-Variables.swift
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

var myMutable: Int = 10;
let myImmutable: Int = 19;

print("myMutable: \(myMutable) myImmutable: \(myImmutable)");

myMutable += myImmutable;

print("myMutable: \(myMutable) myImmutable: \(myImmutable)");
